TOC.md

```diff
@@ -2,187 +2,209 @@
-    - [Configuration File Description](op-guide/tidb-config-file.md)
-    - [Modify Component Configuration Using Ansible](op-guide/ansible-deployment-rolling-update.md#modify-component-configuration)
-    - [Enable TLS Authentication](op-guide/security.md)
++ [Introduction](QUICKSTART.md)
++ Concepts
+  - [Architecture](architecture.md)
+  + [Key Features](overview.md)
+    - [Horizontal Scalability](overview.md#horizontal-scalability)
+    - [MySQL Compatible Syntax](overview.md#mysql-compatible-syntax)
+    - [Replicate from and to MySQL](overview.md#replicate-from-and-to-mysql)
+    - [Distributed Transactions with Strong Consistency](overview.md#distributed-transactions-with-strong-consistency)
+    - [Cloud Native Architecture](overview.md#cloud-native-architecture)
+    - [Minimize ETL with HTAP](overview.md#minimize-etl-with-htap)
+    - [Fault Tolerance & Recovery with Raft](overview.md#fault-tolerance--recovery-with-raft)
+    - [Automatic Rebalancing](overview.md#automatic-rebalancing)
+    - [Deployment and Orchestration with Ansible, Kubernetes, Docker](overview.md#deployment-and-orchestration-with-ansible-kubernetes-docker)
+    - [JSON Support](overview.md#json-support)
+    - [Spark Integration](overview.md#spark-integration)
+    - [Read Historical Data without Restoring from Backup](overview.md#read-historical-data-without-restoring-from-backup)
+    - [Fast Import and Restore of Data](overview.md#fast-import-and-restore-of-data)
+    - [Hybrid of Column and Row Storage](overview.md#hybrid-of-column-and-row-storage)
+    - [SQL Plan Management](overview.md#sql-plan-management)
+    - [Open Source](overview.md#open-source)
+    - [Online Schema Changes](overview.md#online-schema-changes)
++ How-to
+  + Get Started
+    + Start a Local Cluster
+      - [From Binary](op-guide/binary-local-deployment.md)
+      - [In Kubernetes](op-guide/kubernetes-local.md)
+      - [In Docker Compose](op-guide/docker-compose.md)
+    - [Explore SQL with TiDB](try-tidb.md)
+    - [Import Sample Database](bikeshare-example-database.md)
+    - [Read Historical Data](op-guide/history-read.md)
+  - Deploy
+    - [Hardware Recommendations](op-guide/recommendation.md)
+    + From Binary Tarball
+      - [For testing environments](op-guide/binary-testing-deployment.md)
+      - [For production environments](op-guide/binary-deployment.md)
+    + Orchestrated Deployment
+      - [Ansible Deployment (Recommended)](op-guide/ansible-deployment.md)
+      - [Ansible Offline Deployment](op-guide/offline-ansible-deployment.md)
+      - [Docker Deployment](op-guide/docker-deployment.md)
+      - [Kubernetes Deployment](op-guide/kubernetes.md)
+      - [Overview of Ansible Operations](op-guide/ansible-operation.md)
+    + Geographic Redundancy
+      - [Overview](op-guide/cross-dc-deployment.md)
+      - [Configure Location Awareness](op-guide/location-awareness.md)
+    - [TiSpark](tispark/tispark-quick-start-guide.md)
```

Comment:
Need to be more deployment-oriented here, or else users would not know why they need it.  Maybe "OLAP Functionality" or something like it?
The content needs to be adapted later.

Reply:
If we call it OLAP functionality, it is ambiguous between TiDB and TiSpark. The introduction features already describe TiSpark, and I think this appeals most to users who already use Spark - and will contextually realize it's relevance because it has "Spark" in it :-)

Reply:
Sounds good :) That's a good way to put it.